app/src/main/java/com/github/geohunt/app/model/database/api/Claim.kt

```diff
@@ -33,8 +33,16 @@ interface Claim {
      */
     val time: LocalDateTime
 
+    /**
+     * Distance between the actual location and the submitted one
+     */
```

Comment:
The distance unit should be specified in the documentation (m/dm/hm/km ?).

Also, knowing that `Location` have latitude and longitude stored as doubles, and `Location.distanceTo` returning a double, should we use longs or doubles ?